crates/biome_configuration/tests/invalid/top_level_extraneous_field.json.snap

```diff
@@ -16,6 +15,7 @@ top_level_extraneous_field.json:2:2 deserialize ━━━━━━━━━━
   i Known keys:
   
   - $schema
+  - root
```

Comment:
The `root` field was added so we know when to stop scanning upwards inside `open_project()`.